osc-ui/src/main/java/org/osc/core/broker/view/vc/securitygroup/BindSecurityGroupWindow.java

```diff
@@ -122,6 +124,9 @@ public void submitForm() {
 					Object selectedPolicy = ((ComboBox) selectedService.getItemProperty(PROPERTY_ID_POLICY).getValue())
 							.getValue();
 					Long policyId = selectedPolicy == null ? null : ((PolicyDto) selectedPolicy).getId();
+					// TODO Larkins: Fix UI to receive the set of policies from the User
+					Set<Long> policyIds = new HashSet<>();
```

Comment:
provide warning that modifying is not supported. add a field in org.osc.core.broker.service.dto.VirtualSystemPolicyBindingDto to figure out if multiple policy mapping is supported.

disable the policy field in case of multi policy just like we do when policy mapping is not supported.

make sure the request sends original information...so policies are not dropped